src/wp-includes/script-loader.php

```diff
@@ -1027,6 +1027,18 @@ function wp_default_scripts( $scripts ) {
 	$scripts->set_translations( 'wp-theme-plugin-editor' );
 
 	$scripts->add( 'wp-playlist', "/wp-includes/js/mediaelement/wp-playlist$suffix.js", array( 'wp-util', 'mediaelement' ), false, 1 );
+	did_action( 'init' ) && $scripts->localize(
+		'wp-playlist',
+		'_wpmejsSettings',
+		/**
+		 * Filters the MediaElement configuration settings.
+		 *
+		 * @since 4.4.0
+		 *
+		 * @param array $mejs_settings MediaElement settings array.
+		 */
+		apply_filters( 'mejs_settings', $mejs_settings )
+	);
```

Comment:
I don't think this addition is necessary.
The `wp-playlist` script has a dependency on `mediaelement` to `_wpmejsSettings` should already bu output to the page with the `mediaelement`.

I've tested locally with a playlist shortcode after commenting these lines and it still works on the frontend okay with the player icons still showing - am I missing something?